backend_new/src/main.ts

```diff
@@ -1,8 +1,17 @@
 import { NestFactory } from '@nestjs/core';
+import { SwaggerModule, DocumentBuilder } from '@nestjs/swagger';
 import { AppModule } from './app.module';
 
 async function bootstrap() {
   const app = await NestFactory.create(AppModule);
+  const config = new DocumentBuilder()
+    .setTitle('Bloom API')
+    .setDescription('The API for Bloom')
+    .setVersion('1.0')
```

Comment:
Wondering if we anticipate ever updating this, seems easy to forget

Reply:
I bumped it to version 2.0 lol

but yeah we rarely go into the main.ts file but I'll be back in here at some point to add error handling and a few other things so I'll be able to bump

maybe once this become **the** backend we can add a pr template item thingy to up the version here